bin/download_submission.py

```diff
@@ -8,20 +8,36 @@
 def get_args():
     """Set up command-line interface and get arguments without any flags."""
     parser = argparse.ArgumentParser()
-    parser.add_argument("submission_id", type=str, help="The ID of submission")
+    parser.add_argument("--submission-id", "-s", type=str, required=True, help="The ID of submission")
+    parser.add_argument("--file-type", "-f", type=str, required=True, help="The type of file the submission should be")
 
     return parser.parse_args()
 
 if __name__ == "__main__":
     args = get_args()
     submission_id = args.submission_id
+    file_type = args.file_type
 
     syn = synapseclient.Synapse()
     syn.login(silent=True)
 
     submission = syn.getSubmission(submission_id, downloadLocation=".")
     entity_type = submission["entity"].concreteType
-    if entity_type != "org.sagebionetworks.repo.model.FileEntity":
-        open("dummy.txt", 'w').close()
+    file_path = submission["filePath"]
+    
+    # TODO: Eventually we want to abstract this logic into the `make_invalid_file` function
+    # in model-to-data's `run_docker.py`, and move that out somewhere else.
+    invalid_file = f"INVALID_predictions.{file_type}"
+    error_msg = None
 
-    print(entity_type)
+    if entity_type != "org.sagebionetworks.repo.model.FileEntity":
+        error_msg = (
+            f"Only Files should be submitted. Submission {submission_id} type is: {entity_type}"
+        )
+    elif not submission["filePath"].lower().endswith(file_type):
```

Comment:
is the remaining `.lower()` still needed?

Reply:
Yes, just in case the platform team ever decides to return this metadata in all-caps for some reason.